tests/dragonfly/server_family_test.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import redis
+
+def test_quit(connection):
+    connection.send_command("QUIT")
+    assert connection.read_response() == b'OK'
+
+    with pytest.raises(redis.exceptions.ConnectionError) as e:
```

Comment:
what's the purpose of these 2 lines?

Reply:
The upper two?
To send a QUIT and check it actually responds OK...?
The lower one - the most reliable way to check redis is really discontented - try sending something down the connection. Without the fix, this code would hang

Reply:
ok. why would this hang btw? where does it block?

Reply:
In the last read response. Because this is low level, we can manually read the connection even if we don't expect anything. Without the fix, the connection is still open, so it waits